.circleci/unittest/linux/scripts/run_test.sh

```diff
@@ -6,4 +6,4 @@ eval "$(./conda/bin/conda shell.bash hook)"
 conda activate ./env
 
 python -m torch.utils.collect_env
-pytest --cov=torchtext --junitxml=test-results/junit.xml -v --durations 20 test
+pytest --tx "3*popen//python=python" --cov=torchtext --junitxml=test-results/junit.xml -v --durations 20 test
```

Comment:
Can you mention in your summary that this command is creating three subprocesses to run tests in parallel. 

Also, I think it's worth benchmarking what the performance speedups look like for the test suite. Could you run the test suite locally and paste the results before and after parallelizing tests?

Reply:
Yes, absolutely. I can take a crack at something resembling a benchmark.

As an aside, I actually don't like this `--tx ...` idiom very much as it feels very low level and would prefer to use `-n auto` but ran into some errors about how tests get collected.

Reply:
Yeah the `-n auto` flag does seem cleaner since it scales to the number of available CPUs on the host machine. What did your errors look like? Also I would prefer to pass in 4 instead of 3 to the `--tx` command since the number of cores on a machine is usually a multiple of 2.

Reply:
The errors were somewhat non-descript, but generally looked like:

```
Different tests were collected between gw0 and gw1. The difference is: 

--- gw0 
+++ gw1
```

Regarding the multiples-of-2: I believe (but may be mistaken) that the main test process constitutes the 4th process, hence the somewhat ugly 3 processes. It may be a moot point if I can get the `-n auto` working... More info to follow.

Reply:
Ahh gotcha thanks for letting me know! Btw @mthrok strongly suggested not to use `pytest-xdist` and mentioned he would follow up with the reasons later today. Let's try to hold off on this effort until he get's back to us.